releases/sui/2025-01-ITS-v1.1.0.md

```diff
@@ -0,0 +1,152 @@
+MNEMONIC=<wasm-deployer-key>
+PRIVATE_KEY=<sui-deployer-key>
+PRIVATE_KEY_TYPE="mnemonic" # Optional
+SIGNATURE_SCHEME=secp256k1
+ENV=<devnet-amplifier|stagenet|testnet|mainnet>
+CHAIN=sui
+```
+
+### Deploy move contracts
+- Ensure that there isn’t a move directory in Sui directory, if it exists delete it.
+```bash
+rm -rf ./sui/move
+```
+
+- Deploy the ITS Contract and required depndencies on Sui (remember to save `sui/move` files for ITS deployment)
+
+```bash
+node sui/deploy-contract.js deploy Abi
+node sui/deploy-contract.js deploy InterchainTokenService
+node sui/deploy-contract.js deploy Example
+```
+
+### Register Sui chain on core
+
+```bash
+ITS_HUB_ADDR=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.axelar.contracts.InterchainTokenService.address')
+ITS_SUI_CHANNEL=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.chains.sui.contracts.InterchainTokenService.objects.ChannelId')
+```
+
+#### Devnet:
+```bash
+kubectl exec genesis-0 -- axelard tx wasm execute "$ITS_HUB_ADDR" \
+'{
+  "register_chains": {
+    "chains": [
+      {
+        "chain": "sui",
+        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
+        "truncation": {
+          "max_decimals_when_truncating": '"8"',
+          "max_uint": "'"18446744073709551615"'"
+        }
+      }
+    ]
+  }
+}' \
+--from <name-of-your-wasm-deployer-key> \
+--gas auto \
+--gas-adjustment 1.2
+```
+
+#### Stagenet/Testnet:
+```bash
+node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d "Register ITS for Sui chain at ITS Hub contract" --deposit <100000000 for stagenet and 2000000000 for testnet> \
+-m '{
+  "register_chains": {
+    "chains": [
+      {
+        "chain": "sui",
+        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
+        "truncation": {
+          "max_decimals_when_truncating": '"8"',
+          "max_uint": "'"18446744073709551615"'"
+        }
+      }
+    ]
+  }
+}'
+```
+
+#### Mainnet:
+```bash
+node cosmwasm/submit-proposal.js its-hub-register-chains sui -t "Register ITS for Sui chain" -d <Description along with community proposal for ITS integration> --deposit 2000000000 \
+-m '{
+  "register_chains": {
+    "chains": [
+      {
+        "chain": "sui",
+        "its_edge_contract": "'"$ITS_SUI_CHANNEL"'",
+        "truncation": {
+          "max_decimals_when_truncating": '"8"',
+          "max_uint": "'"18446744073709551615"'"
+        }
+      }
+    ]
+  }
+}'
+```
+
+### Setting up trusted addresses
+```bash
+ITS_HUB_ADDR=$(cat ./axelar-chains-config/info/<your_config_name>.json | jq -rM '.axelar.contracts.InterchainTokenService.address')
+
+node sui/its.js add-trusted-chains axelar $ITS_HUB_ADDR
+
+# comma separated list of existing chains of ITS
+node evm/its.js -n avalanche-fuji,ethereum-sepolia --action setTrustedAddress --trustedChain sui --trustedAddress hub
+
+# the other way around, done for all existing chains with ITS
+node sui/its.js add-trusted-chains all-evm
```

Comment:
```suggestion
node sui/its.js add-trusted-chains all
```
Should be all, and not just EVM. For e.g. if Stellar ITS is deployed, it should be listed as well. `all-evm` isn't particularly useful